openshift/job-dependencyMonkey-template.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     thoth-station.ninja/template-version: 0.4.5
```

Comment:
pls incr this

Reply:
Done in 4d489584b0ab2882c8b6a4c27974f9a771b5ea0c